sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/view/projects/ProjectFilter.java

```diff
@@ -0,0 +1,46 @@
+package life.qbic.portal.sampletracking.view.projects;
+
+import java.util.Objects;
+import life.qbic.portal.sampletracking.view.GridFilter;
+import life.qbic.portal.sampletracking.view.projects.viewmodel.Project;
+
+/**
+ * <b>short description</b>
+ *
+ * <p>detailed description</p>
+ *
+ * @since <version tag>
+ */
+public class ProjectFilter implements GridFilter<Project> {
+
+  private String containedText = "";
+
+  public ProjectFilter containingText(String substring) {
+    this.containedText = substring;
+    return this;
+  }
+
+
+  @Override
+  public boolean test(Project project) {
```

Comment:
call it something with "filter"?

Reply:
With filter I would suspect to get null or the object back instead of a boolean. Same with filter.apply